Exec/Production/ChallengeProblem/challenge.inp

```diff
@@ -52,22 +50,21 @@ amr.data_log         = datlog extremalog
 amr.max_level       = 5       # maximum level number allowed
 amr.ref_ratio       = 2 2 2 2 2 2 # refinement ratio
 amr.regrid_int      = 2       # how often to regrid
-amr.blocking_factor = 8       # block factor in grid generation
-amr.max_grid_size   = 16
+amr.blocking_factor = 16       # block factor in grid generation
+amr.max_grid_size   = 64
 amr.n_error_buf     = 2 2 2 2 2 2 # number of buffer cells in error est
-#amr.hydro_tile_size = 8 8 8
 
 # LOAD BALANCE
 amr.loadbalance_with_workestimates = 1
 #amr.loadbalance_level0_int = 1000
 
 # CHECKPOINT FILES
-amr.checkpoint_files_output = 1
+amr.checkpoint_files_output = 0
```

Comment:
hmmm I get that we want these for profiling. But we are going to want chk and plt out of this.

Reply:
I can put these back.